webview/src/setup/components/studio/styles.module.scss

```diff
@@ -0,0 +1,3 @@
+.smallFont {
+  font-size: 0.6rem;
```

Comment:
I don't have the best eyesight, but I had to squint a bit to read this 🤣. Should we increase it a bit? Example of `0.7rem`: 

<img width="391" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/e8486cb0-f327-4f61-85fc-a8b013d6f3c6">